modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/shared/ClaySpinner.es.js

```diff
@@ -20,12 +20,12 @@ class ClaySpinner extends Component {
 			}
 		);
 
-		return loading && (
+		return loading ? (
```

Comment:
How come you switched to a ternary here instead? Both should evaluate to the same thing.

Reply:
I did it because a render method should never return `undefined`.
https://reactjs.org/docs/react-component.html#render

Reply:
In that case, we should be setting a default value for "loading" since we know it will and always should be a boolean. In that case, loading && ... will always return a boolean or jsx.